.travis.yml

```diff
@@ -370,7 +370,9 @@ jobs:
       env:
         - PROJECT=GoogleUtilities METHOD=pod-lib-lint
       script:
```

Comment:
Looks good. Please make the same change for the GoogleUtilitiesCron job below.

Reply:
Done. There are two cmds under GoogleUtilitiesCron so I split each of them to 3 platforms.